eng/pipelines/libraries/build-for-test-ready-to-run.yml

```diff
@@ -0,0 +1,41 @@
+parameters:
+  archType: ''
+  container: ''
+  buildConfig: ''
+  crossBuild: false
+  dependOnEvaluatePaths: false
+  helixQueues: ''
+  osGroup: ''
+  osSubgroup: ''
+  platform: ''
+  pool: ''
+  runtimeFlavor: 'coreclr'
+  shouldContinueOnError: false
+  targetRid: ''
+  variables: []
+
+jobs:
+  - template: /eng/pipelines/common/global-build-job.yml
```

Comment:
I may be mistaken but I think that "global-build-job.yml" is the pipeline that's supposed to test "local dev experience" in terms of building and running everything in sequence on the same machine. Is that what we really need here? I would assume that we should be able to reuse the library build artifacts and possibly just build the live CoreCLR subrepo including Crossgen2, possibly along with product.packs in case we want to use the "shipping NativeAOT Crossgen2" for the testing.

Reply:
We've been moving to using the `global-build-job.yml` template as the "build the repo with the root build script" command instead of using the separate CoreCLR, Libraries, etc build jobs for pipelines other than the runtime pipeline.